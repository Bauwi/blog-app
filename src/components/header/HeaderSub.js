import React from 'react';
import { NavLink } from 'react-router-dom';

const HeaderSub = () => (
  <nav className="header__navbarsub">
    <NavLink to="/category/architecture" activeClassName="architecture">
      architecture
    </NavLink>
    <NavLink to="/category/sculpture" activeClassName="sculpture">
      scuplture
    </NavLink>
    <NavLink to="/category/visuals" activeClassName="visuals">
      visuals
    </NavLink>
    <NavLink to="/category/music" activeClassName="music">
      music
    </NavLink>
    <NavLink to="/category/literature" activeClassName="literature">
      literature
    </NavLink>
    <NavLink to="/category/performings" activeClassName="performings">
      performings
    </NavLink>
    <NavLink to="/category/cinema" activeClassName="cinema">
      cinema
    </NavLink>
    <NavLink to="/category/photography" activeClassName="photography">
      photography
    </NavLink>
  </nav>
);

export default HeaderSub;
