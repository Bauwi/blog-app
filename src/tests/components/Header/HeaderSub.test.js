import React from 'react';
import { shallow } from 'enzyme';
import HeaderSub from '../../../components/header/HeaderSub';

test('should render HeaderSub correctly', () => {
  const wrapper = shallow(<HeaderSub />);

  expect(wrapper).toMatchSnapshot();
});
