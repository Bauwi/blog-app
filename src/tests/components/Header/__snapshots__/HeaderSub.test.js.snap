// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HeaderSub correctly 1`] = `
<nav
  className="header__navbarsub"
>
  <NavLink
    activeClassName="header__navbarsub__tab--selected"
    ariaCurrent="true"
    to="/category/architecture"
  >
    architecture
  </NavLink>
  <NavLink
    activeClassName="header__navbarsub__tab--selected"
    ariaCurrent="true"
    to="/category/sculpture"
  >
    scuplture
  </NavLink>
  <NavLink
    activeClassName="header__navbarsub__tab--selected"
    ariaCurrent="true"
    to="/category/visuals"
  >
    visuals
  </NavLink>
  <NavLink
    activeClassName="header__navbarsub__tab--selected"
    ariaCurrent="true"
    to="/category/music"
  >
    music
  </NavLink>
  <NavLink
    activeClassName="header__navbarsub__tab--selected"
    ariaCurrent="true"
    to="/category/literature"
  >
    literature
  </NavLink>
  <NavLink
    activeClassName="header__navbarsub__tab--selected"
    ariaCurrent="true"
    to="/category/performings"
  >
    performings
  </NavLink>
  <NavLink
    activeClassName="header__navbarsub__tab--selected"
    ariaCurrent="true"
    to="/category/cinema"
  >
    cinema
  </NavLink>
  <NavLink
    activeClassName="header__navbarsub__tab--selected"
    ariaCurrent="true"
    to="/category/photography"
  >
    photography
  </NavLink>
</nav>
`;
